src/sys/socket/sockopt.rs

```diff
@@ -270,6 +270,16 @@ sockopt_impl!(
     libc::SO_REUSEPORT,
     bool
 );
+#[cfg(target_os = "freebsd")]
```

Comment:
Any chance that we can add a unit test for this option? Maybe a simple set-and-get test?